### PR TITLE
refactor: apply updates of text-input, filterable-dropdown

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

- Applied updates of text-input
- Applied updates of filterable-dropdown

### Things to Talk About

Please look ProjectMemberAddModal.vue carefully

### CODE REVIEW

File: src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue [hunk 1]
```diff
@@ -204,45 +221,50 @@ export default {
...
+            const errorCodes = union(Object.values(externalItemsErrors.value)).filter((code) => !!code);
+            return errorCodes.map((code) => {
+                if (code === 'DUPLICATED') return i18n.t('PROJECT.DETAIL.MEMBER.DUPLICATED_VALUE');
+                if (code === 'EMAIL_FORMAT') return i18n.t('IDENTITY.USER.FORM.EMAIL_INVALID');
+                if (code === 'ALREADY_EXIST') return i18n.t('PROJECT.DETAIL.MEMBER.ALREADY_EXISTING');
+                if (code === 'NOT_FOUND') return i18n.t('PROJECT.DETAIL.MEMBER.INVALID');
+                return undefined;
+            });
+        });
+        const selectedExternalUserItems = ref<InputItem[]>([]);
```
Comment (yuda110) [inline src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue:245]: 👀👀👀👀

File: src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue [hunk 2]
```diff
@@ -289,28 +311,25 @@ export default {
...
-            }
-
-            /* 2. check email validation */
-            const emailFormValidation = checkEmailFormat(userItem.value);
+        const setExternalItemsInvalidTexts = async (userItem: InputItem): Promise<ExternalItemErrorCode|undefined> => {
+            /* 1. check email validation */
+            const emailFormValidation = checkEmailFormat(userItem.name ?? '');
             if (!emailFormValidation.isValid) {
-                return emailFormValidation.invalidText;
+                return 'EMAIL_FORMAT';
```
Comment (yuda110) [inline src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue:318]: praise: 👍

**Review by yuda110 — APPROVED** [review]
LGTM...! Nice refactoring!!!!

File: src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue (hunk 1 above)
Comment (piggggggggy) [reply src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue:245]: 👀👀👀👀

File: src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue [hunk 3]
```diff
@@ -5,7 +5,7 @@
         :fade="true"
         :backdrop="true"
         :visible.sync="proxyVisible"
-        :disabled="!isAllValid"
+        :disabled="!isAllValid && !isLabelDuplicated && !externalItemsInvalidTexts.length"
```
Comment (piggggggggy) [inline src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue:8]: question: Perhaps... code below is correct?
```suggestion
        :disabled="!isAllValid || isLabelDuplicated || externalItemsInvalidTexts.length"
```

**Review by piggggggggy — APPROVED** [review]
LGTM!!
It's so hard work!!

I left a small comment!

File: src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue (hunk 3 above)
Comment (WANZARGEN) [reply src/services/project/project-detail/project-member/modules/ProjectMemberAddModal.vue:8]: Oh thanx!!

**Pull request merged**